notes/emergency.vbs

```diff
@@ -235,26 +236,26 @@ If EGA_screening_check = 1 then
     End if
 
     'determining  200% FPG per HH member---handles up to 20 members. Changes April 1 every year. CM0016.18.01 - 200 Percent of FPG
-	If HH_members = "1"  then monthly_standard = "2265"
-    If HH_members = "2"  then monthly_standard = "3052"
-    If HH_members = "3"  then monthly_standard = "3838"
-    If HH_members = "4"  then monthly_standard = "4625"
-    If HH_members = "5"  then monthly_standard = "5412"
-    If HH_members = "6"  then monthly_standard = "6198"
-    If HH_members = "7"  then monthly_standard = "6985"
-    If HH_members = "8"  then monthly_standard = "7772"
-    If HH_members = "9"  then monthly_standard = "8558"
-    If HH_members = "10" then monthly_standard = "9345"
-    If HH_members = "11" then monthly_standard = "10132"
-    If HH_members = "12" then monthly_standard = "10918"
-    If HH_members = "13" then monthly_standard = "11705"
-    If HH_members = "14" then monthly_standard = "12492"
-    If HH_members = "15" then monthly_standard = "13278"
-    If HH_members = "16" then monthly_standard = "14065"
-    If HH_members = "17" then monthly_standard = "14852"
-    If HH_members = "18" then monthly_standard = "15638"
-    If HH_members = "19" then monthly_standard = "16425"
-    If HH_members = "20" then monthly_standard = "17211"
+	If HH_members = "1"  then monthly_standard = "2430"
+    If HH_members = "2"  then monthly_standard = "3287"
+    If HH_members = "3"  then monthly_standard = "4143"
+    If HH_members = "4"  then monthly_standard = "5000"
+    If HH_members = "5"  then monthly_standard = "5857"
+    If HH_members = "6"  then monthly_standard = "6713"
+    If HH_members = "7"  then monthly_standard = "7570"
+    If HH_members = "8"  then monthly_standard = "8427"
+    If HH_members = "9"  then monthly_standard = "9283"
+    If HH_members = "10" then monthly_standard = "10140"
+    If HH_members = "11" then monthly_standard = "10997"
```

Comment:
It's weird that we aren't using the calculation for HH size over 10... but it would also be weird to have a HH size over 2 for this script. Don't think we need to change it here but we could update this to make the update easier in future years.

Reply:
@MDR22 @MeganGeissler - what do you think - make sense?

Reply:
Looks great!

Reply:
Agreed @C-Love it's a bit clunky. This script has a lot of opportunities for improvement.